src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,84 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1Files = (files) => {
+    const fileNames = [];
+    const apiV1Files = {};
+
+    Object.keys(files).forEach((key) => {
+      const fileType = files[key].sampleFileType;
+      let fileName = '';
+      if (fileType.includes('features')) {
+        fileName = 'features.tsv.gz';
+      } else if (fileType.includes('barcodes')) {
+        fileName = 'barcodes.tsv.gz';
+      } else if (fileType.includes('matrix')) {
```

Comment:
Rather than `includes` I would compare directly, we know exactly what types are possible: `features10x`, `barcodes10x` and `matrix10x`